README.md

```diff
@@ -343,315 +375,340 @@ convert asff2hdf            Translate a AWS Security Finding Format JSON into a
 
-  EXAMPLES
-    saf convert sonarqube2hdf -n project_key -u http://sonar:9000 --auth YOUR_API_KEY -o output-hdf-name.json
+  FLAGS
+    -a, --auth=<value>           (required) SonarQube API Key
+    -b, --branch=<value>         Requires Sonarqube Developer Edition or above
+    -h, --help                   Show CLI help.
+    -n, --projectKey=<value>     (required) SonarQube Project Key
+    -o, --output=<value>         (required) Output HDF JSON File
+    -p, --pullRequestID=<value>  Requires Sonarqube Developer Edition or above
+    -u, --url=<value>            (required) SonarQube Base URL (excluding '/api')
 
+  EXAMPLES
+    $ saf convert sonarqube2hdf -n sonar_project_key -u http://sonar:9000 --auth YOUR_API_KEY [ -b my_branch || -p 123 ]-o scan_results.json
 ```
 
 ##### Splunk to HDF
 ```
-convert splunk2hdf           Pull HDF data from your Splunk instance back into an HDF file
+convert splunk2hdf            Pull HDF data from your Splunk instance back into an HDF file
 
-USAGE
-  $ saf splunk2hdf -i, --input=FILE -H, --host -P, --port -p, --protocol -t, --token -i, --index
-
-FLAGS
-  -H, --host=<value>       (required) Splunk Hostname or IP
-  -I, --index=<value>      (required) Splunk index to query HDF data from
-  -L, --logLevel=<option>  [default: info]
-                           <options: info|warn|debug|verbose>
-  -P, --port=<value>       [default: 8089] Splunk management port (also known as the Universal Forwarder port)
-  -i, --input=<value>...   GUID(s) or Filename(s) of files to convert
-  -o, --output=<value>     Output HDF JSON Folder
-  -p, --password=<value>   Your Splunk password
-  -s, --scheme=<option>    [default: https] HTTP Scheme used for communication with splunk
-                           <options: http|https>
-  -t, --token=<value>      Your Splunk API Token
-  -u, --username=<value>   Your Splunk username
+  USAGE
+    $ saf splunk2hdf -H <host> -I <index> [-h] [-P <port>] [-s http|https] (-u <username> -p <password> | -t <token>) [-L info|warn|debug|verbose] [-i <filename/GUID> -o <hdf-output-folder>]
 
-EXAMPLES
-  saf convert splunk2hdf -H 127.0.0.1 -u admin -p Valid_password! -I hdf -i some-file-in-your-splunk-instance.json yBNxQsE1mi4f3mkjtpap5YxNTttpeG -o output-folder
-  saf convert splunk2hdf -I hdf -H 127.0.0.1 -t your.splunk.token
-```
+  FLAGS
+    -H, --host=<value>       (required) Splunk Hostname or IP
+    -I, --index=<value>      (required) Splunk index to query HDF data from
+    -L, --logLevel=<option>  [default: info]
+                            <options: info|warn|debug|verbose>
+    -P, --port=<value>       [default: 8089] Splunk management port (also known as the Universal Forwarder port)
+    -h, --help               Show CLI help.
+    -i, --input=<value>...   GUID(s) or Filename(s) of files to convert
```

Comment:
make it clearer that this is a filename from splunk, not a local file like it is for basically every other -i we got